python-package/securexgboost/core.py

```diff
@@ -1502,7 +1538,7 @@ def eval(self, data, name='eval', iteration=0):
 
     def predict(self, data, output_margin=False, ntree_limit=0, pred_leaf=False,
                 pred_contribs=False, approx_contribs=False, pred_interactions=False,
-                validate_features=True):
+                validate_features=True, username = None):
```

Comment:
Please change this to `username=None`, i.e. without the spaces

Reply:
done